app/models/ownership.rb

```diff
@@ -3,17 +3,4 @@ class Ownership < ActiveRecord::Base
   belongs_to :user
 
   validates :user_id, uniqueness: { scope: :rubygem_id }
-
-  before_destroy :keep_last_owner
```

Comment:
Yep, IIRC It's in place so someone can't remove themselves and orphan the gem - not so users can't remove others. That's definitely OK to do.

We definitely want to use the `Deletion` model to remove these ownerships so we have a record of them when the user being deleted is the only owner.

There's a lot of cases here to cover 😰 

Re: that check in the `OwnersController`, that kind of check should definitely happen in the model instead.